### PR TITLE
Bc gov look and feel

### PR DESCRIPTION
**Pull request by ShovelHand**
I'm not sure what happened to the last pull request. Patrick, I wonder if you would give this a quick once over before I merge this branch into main, and move on to other sprint goals.

### CODE REVIEW

File: build/report.py [hunk 1]
```diff
@@ -36,6 +36,25 @@ def getServer():
   return server
 #end
 
+def getCluster():
```
Comment (patricksimonian) [inline build/report.py:39]: i didn't even know how to do this

File: build/static/style.css [hunk 2]
```diff
@@ -3,19 +3,42 @@
...
 }
-.green {
-  background-color: green;
+.pass{
+  background-color: #2E8540;
+  border-color: #2E8540;
 }
-.yellow {
-  background-color: yellow;
+.warning{
```
Comment (patricksimonian) [inline build/static/style.css:22]:
```suggestion
.warning {
```

File: build/static/style.css [hunk 3]
```diff
@@ -3,19 +3,42 @@
...
+  display: inline-grid;
+}
+
+.table-legend {
+  margin-left: 10%;
+  max-width: 50%;
+}
+.table-header-label {
+  text-align: center;
+}
```
Comment (patricksimonian) [inline build/static/style.css:44]: new line

File: build/templates/reportTemplate.html.j2 [hunk 4]
```diff
@@ -12,48 +12,122 @@
             <!-- Navbar content -->
             <div class="container">
                <a class="navbar-brand" href="https://www2.gov.bc.ca">
-                  <img class="img-fluid d-none d-md-block" src="https://developer.gov.bc.ca/static/BCID_H_rgb_rev-20eebe74aef7d92e02732a18b6aa6bbb.svg" width="177" height="44" alt="B.C. Government Logo">
+               <img class="img-fluid d-none d-md-block" src="https://developer.gov.bc.ca/static/BCID_H_rgb_rev-20eebe74aef7d92e02732a18b6aa6bbb.svg" width="177" height="44" alt="B.C. Government Logo">
```
Comment (patricksimonian) [inline build/templates/reportTemplate.html.j2:15]: should be indented

File: build/templates/reportTemplate.html.j2 [hunk 5]
```diff
@@ -98,4 +189,4 @@
          </nav>
       </footer>
    </body>
-</html>
+</html>
```
Comment (patricksimonian) [inline build/templates/reportTemplate.html.j2:192]: new line

**Review by patricksimonian — APPROVED** [review]

File: build/report.py (hunk 1 above)
Comment (ShovelHand) [reply build/report.py:39]: There was no cool trick or API call or anything. I just had a look at the JSON returned by getServer, and changed the return value to get the cluster instead.

**Pull request merged**